lib/tasks/build.rake

```diff
@@ -1,5 +1,98 @@
 require 'zip_file_generator'
 
+namespace :build do
+  namespace :omninav do
+    #
+    # Build static version: rake build:omninav:static
+    #
+    desc "Build static OmniNav navbar assets and markup."
+    task static: :environment do
+      # Params
+      target = 'static'
+
+      # Prep Builder
+      builder = Omninav::Builder.new(target: target)
+      builder.prep_build
+      builder.deploy_map = {
+        'omni-nav-*.css' => 'omni-nav.min.css',
+        'omni-nav-*.js' => 'omni-nav.min.js',
+        'omni-nav-*.css.gz' => 'omni-nav.css.gz',
+        'omni-nav-*.js.gz' => 'omni-nav.js.gz',
+        'omni-nav.html' => 'omni-nav.html'
+      }
+      puts format("\nBuilding OmniNav %s version:%s\n", target, builder.build_version)
+
+      # Why this? Setting Rails.env or ENV['RAILS_ENV'] didn't work.
+      # See http://stackoverflow.com/a/1090319/6763239.
+      system("rake build:omninav:assets RAILS_ENV=production")
```

Comment:
This is how I finally got the rake task to minify assets. That's my StackOverflow question.